docs/build-tools/gradle.md

```diff
@@ -34,20 +30,16 @@ buildscript {
 }
 ```
 
-Secondly, we need to enable the plugin for all the projects we want to include,
-it's easiest to define it for `allprojects`:
+Secondly, we need to enable the plugin for all the projects we want to include.
+It's easiest to define it for `allprojects`:
 
 ```groovy
 allprojects {
    apply plugin: bloop.integrations.gradle.BloopPlugin
 }
 ```
 
-Now we can run `gradle bloopInstall`, which will create a Bloop configuration
-files.
+Now we can run `gradle bloopInstall`, which will create all of the Bloop
+configuration files.
 
 This will enable us to work with Metals and all features should work.
-
-You can also always try to reach us on the
-[Metals gitter channel](https://gitter.im/scalameta/metals) in case of any
```

Comment:
Why remove it?

Reply:
Mainly for consistency. We don't have it on any of the other build tool pages, so we should either have it on all or none. Plus, this is also in the footer.

Reply:
Ok, it's fine with me then!